src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -215,10 +217,14 @@ class AuthScreens extends React.Component {
         };
         const fullscreenModalScreenOptions = {
             ...commonModalScreenOptions,
-            cardStyle: {...styles.fullscreenCard},
+            cardStyle: {
+                ...styles.fullscreenCard,
+                padding: this.props.isSmallScreenWidth ? spacing.p0.padding : spacing.p5.padding,
+            },
             cardStyleInterpolator: props => modalCardStyleInterpolator(this.props.isSmallScreenWidth, true, props),
-            cardOverlayEnabled: false,
+            cardOverlayEnabled: !this.props.isSmallScreenWidth,
             isFullScreenModal: true,
+            cardOverlay: CardOverlay,
```

Comment:
Sould it make more sense just to have

```js
cardOverlay: <View style={styles.cardOverlay} />,
```

Why have an entirely different file for something that is only used in this one place?

Reply:
We could move it to the AuthScreen but, we would still need to pass it as `cardOverlay: CardOverlay`. I think it's best to leave it as a separate file just in case we need to use it for another modal.